compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/SuspendingPointerInputFilter.kt

```diff
@@ -738,42 +738,17 @@ internal class SuspendingPointerInputModifierNodeImpl(
     }
 }
 
-/**
- * An exception thrown from [AwaitPointerEventScope.withTimeout] when the execution time
- * of the coroutine is too long.
- */
-class PointerEventTimeoutCancellationException(
-    time: Long
-) : CancellationException("Timed out waiting for $time ms") {
-    override fun fillInStackTrace(): Throwable {
-        // Avoid null.clone() on Android <= 6.0 when accessing stackTrace
-        stackTrace = emptyArray()
-        return this
-    }
-}
```

Comment:
Moved public class `PointerEventTimeoutCancellationException` to another file
It will save binary backward compatibility?

Reply:
It has the same package and class name

Reply:
It should be ok for classes